docs/releasenotes/1.3_migration_guide.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: default
+title: v1.3.0 Migration Guide
+parent: pages.release_notes
+nav_order: 1
+---
+
+# v1.2.0 Migration Guide
+{: .no_toc }
+
+## Table of contents
+{: .no_toc .text-delta }
+
+1. TOC
+{:toc}
+
+---
+
+## Overview
+
+Hyperledger FireFly v1.3.0 is a feature release that includes changes around event streaming, contract listeners, define/publish APIs as well as a range of general fixes.
+
+For the most part, upgrading from v1.2.x to v1.3.0 should be a seamless experience, but there are several important things to note about changes between the two versions, which are described in detail on this page.
+
+## API definition/publication considerations
+
+**As of FireFly v1.3.0, by default, contract interfaces, contracts APIs, and token pools are unpublished when they are created.**
+
+These following described changes impact contract interfaces, contract APIs, and token pools. Previously, when creating one of the affected resources, if successful, the resource would be automatically peered to other members of the multi-party network. This default behaviour has changed, now when one of those resources is defined, it by default is not peered to other members of the network, unless a publish query parameter has been provided. 
```

Comment:
```suggestion
These following described changes impact contract interfaces, contract APIs, and token pools. Previously, when creating one of the affected resources, if successful, the resource would be automatically broadcasted to other members of the multi-party network. This default behaviour has changed, now when one of those resources is defined, it by default is not broadcasted to other members of the network, unless a publish query parameter has been provided. 
```

Reply:
Resolved in https://github.com/hyperledger/firefly/pull/1461/commits/e78839f3382b6e0bc379baa9b88331c60e303f94